src/sqlancer/postgres/gen/PostgresAlterTableGenerator.java

```diff
@@ -98,6 +98,20 @@ public List<Action> getActions(ExpectedErrors errors) {
             // make it more likely that the ALTER TABLE succeeds
             action = Randomly.subset(Randomly.smallNumber(), Action.values());
         }
+
+        // If this is a view, only allow view-compatible operations
+        if (randomTable.isView()) {
+            // Remove all table-specific operations for views
+            action.removeIf(a -> a != Action.ALTER_VIEW_RENAME_COLUMN);
```

Comment:
Do we want to create a list of actions (that are only possible on a view) and use that list in this check? That way, if / when we add more view-only actions, we wouldn't miss them here (and line 112 below).